src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1211,6 +1211,54 @@ key is not present, no replacement occurs):
 </task>
 ----
 
+== Fork environment
+A fork execution environment is a new java virtual machine which is started exclusively to execute a task. Starting a
```

Comment:
I would have replaced _java virtual machine_ by _Java Virtual Machine (JVM)_ for the first occurrence and then by _JVM_ for all other occurrences.